pkgs/cli/src/main.zig

```diff
@@ -268,6 +274,61 @@ pub fn main() !void {
             try beam_node_2.run();
             try clock.run();
         },
+        .sim => |simcmd| {
```

Comment:
we don't need this command, current beam command is already a 2 (out of 3 node) sim

point of this entire PR was to actually start the sim, actually test that the node is up, actually make an http request to metrics endpoint and see if we can get the metrics etc